components/dashboard/src/admin/WorkspacesSearch.tsx

```diff
@@ -73,9 +79,9 @@ export function WorkspaceSearch(props: Props) {
 
     const search = async () => {
         // Disables empty search on the workspace search page
-        if (!props.user && queryTerm.length === 0) {
-            return;
-        }
+        // if (!props.user && queryTerm.length === 0) {
+        //     return;
+        // }
```

Comment:
I believe this was the cause of an incident when we originally added the WS search - it would cause too many queries (but may be okay now)

Reply:
Yes, was this maybe before we introduced limits? cc @geropl Do you remember?
It's for sure a SaaS hack that makes the view less useful in smaller self-hosted installations. If we need to protect us we better do that on the server based on a saas-specific configuration.